test/codegen_test.rb

```diff
@@ -73,6 +73,18 @@ def test_generate_types
       assert_equal File.read("test/fixtures/typed_test.correct.rb"), gen.to_ruby
     end
 
+    def test_ruby_package
+      fixture = "test/fixtures/ruby_package_test"
+
+      proto = File.read("#{fixture}.proto")
+      unit = ProtoBoeuf.parse_string(proto)
+
+      gen = CodeGen.new unit, generate_types: true
+
+      File.write("#{fixture}.generated.rb", gen.to_ruby)
+      assert_equal File.read("#{fixture}.correct.rb"), gen.to_ruby
```

Comment:
I think this test is too rigid. It will break if we change the codegen in any way. Ideally you should test for the specific module names you are looking for.

Also, it seems that you are writing the generated code to a file for no reason?